.storybook/main.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/addon-postcss"],
```

Comment:
**question** I guess this has nothing to do with the auth? What is it about?

Reply:
It is related to a deprecated postcss https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-default-postcss-plugins
Which is not yet our case but I had some weird errors on build storybook and I tried more things. This helped somehow.